app/src/main/java/org/p2p/wallet/home/ui/container/MainContainerFragment.kt

```diff
@@ -226,7 +213,8 @@ class MainContainerFragment :
             }
             else -> {
                 when (clickedTab) {
-                    ScreenTab.MY_CRYPTO_SCREEN -> presenter.logHomeOpened()
+                    ScreenTab.WALLET_SCREEN -> presenter.logWalletOpened()
```

Comment:
are we sure that we want to add some analytic methods to presenter? 
maybe inject analytics just here and remove this proxy methods from presenter?